### ISSUE TITLE
Zammad won't send e-mail notifications via triggers unless inbound e-mail is set up

### ISSUE THREAD

**Issue by lpar** [zammad/zammad#2352]
* Zammad Version: 2.7 
* Used OS: CentOS 7
* Installation-Type: Yum/RPM
* Ticket-ID: #1030868

I expected customers to get notified by e-mail and web when their ticket was updated, but this turned out not to be the case, so I set up a notification e-mail trigger as per the documentation. However, the log shows:

`I, [2018-11-13T17:21:10.187601 #24017] INFO -- : Unable to send trigger based notification to [addresses redacted] because no email address is set for group 'Users'`

The problem is, I can't set an e-mail address for group "Users" (or any other group) because I have no _inbound_ e-mail accounts. I have no IMAP server accessible to the Zammad server, no inbound e-mail support at all, only outbound SMTP.

I can't set up a fake inbound e-mail account either, because Zammad insists on checking it can log in via IMAP before letting me save the configuration.

So in summary: Zammad should not require inbound e-mail and a working IMAP server in order to support outbound SMTP e-mail notifications.

**Comment by MrGeneration** [comment]
Hannes and I think this is a bug - a trigger should fallback to notification E-Mail if neceassry to ensure "notifications" made by a trigger can be send.

This also applies to a customer Ticket: #1030868

**Comment by lpar** [comment]
Thanks. For now I've worked around it by installing an IMAP server and setting up a fake account.

**Comment by MvanRietschoten** [comment]
+1

**Comment by MrGeneration** [comment]
I'm limitting this conversation to contributors only as we haven't received any further relevant feedback upon this issue.

This is to reduce `+1` and similar comments which cause noise on the tracker and to people watching this repository in general.

We are working hard on solving issues step by step and will update this issue accordingly as soon as we have any updates on this topic.
